### PR TITLE
refactor(console): update rbac phrases

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

refactor(console): update rbac phrases

## Testing

Test locally.

### CODE REVIEW

File: packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts [hunk 1]
```diff
@@ -17,13 +17,13 @@ const api_resource_details = {
...
     name: 'Permission name', // UNTRANSLATED
-    name_placeholder: 'Read: Resources', // UNTRANSLATED
+    name_placeholder: 'Read:Resources', // UNTRANSLATED
     description: 'Description', // UNTRANSLATED
     description_placeholder: 'Able to read the resources', // UNTRANSLATED
     permission_created: 'The permission {{name}} has been successfully created', // UNTRANSLATED
     delete_description:
       'If this permission is deleted, the user who had this permission will lose the access granted by it.', // UNTRANSLATED
-    deleted: 'The permission {{name}} has been successfully deleted', // UNTRANSLATED
+    deleted: 'The permission "{{name}}" was successfully deleted!', // UNTRANSLATED
```
Comment (charIeszhao) [inline packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts:26]: Why? I think the previous one is better

File: packages/phrases/src/locales/de/translation/admin-console/user-details.ts [hunk 2]
```diff
@@ -46,8 +46,9 @@ const user_details = {
     name_column: 'Role', // UNTRANSLATED
     description_column: 'Description', // UNTRANSLATED
     assign_button: 'Assign Roles', // UNTRANSLATED
-    delete_description: 'TBD', // UNTRANSLATED
-    deleted: 'The role {{name}} has been successfully deleted from the user.', // UNTRANSLATED
+    delete_description:
+      'This action will removing this role from this user. The role itself will still exist, but it will no longer be associated with this user.', // UNTRANSLATED
```
Comment (charIeszhao) [inline packages/phrases/src/locales/de/translation/admin-console/user-details.ts:50]:
```suggestion
      'This action will remove this role from this user. The role itself will still exist, but no longer be associated with this user.', // UNTRANSLATED
```

**Review by simeng-li — APPROVED** [review]
LGTM

File: packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts:26]: This PR is focused on the phrase update. As for the content, maybe discuss it with @fleuraly offline.

Comment (charIeszhao) [reply packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts:26]: ok, np. let's move on

**Pull request merged**